src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-    <l-map ref="map" :zoom="zoom" :center="center" style="height: 800px; width: 100%">
+    <l-map ref="map" v-bind:center="mapCenter" style="height: 800px; width: 100%">
+        <l-geo-json ref="selectedRegionGeoJson" :geojson="selectedRegionFeature"
```

Comment:
The easiest way to get the bounds from the selected feature seemed to be to render it as a component and then ask for its bounds (hence having to wait for the nextTick in the update method below). You can't assume that it will already be rendered as it may not be at the current selected level. Render this feature before the others so that it is behind, and the tooltips for the current features still work!